.gitignore

```diff
@@ -15,3 +15,5 @@ yarn-error.log
 .docusaurus/
 build/
 types
+package-lock.json
+yarn.lock
```

Comment:
Why did you add these to gitignore?
AFAIK it is important to keep these in the repo to make sure all are using the same dependency versions.

see https://yarnpkg.com/getting-started/qa#should-lockfiles-be-committed-to-the-repository

Reply:
Thanks for the article, last I read about those issues were ages ago and now the lock files are needed to be added. I will do so and push the changes.